packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -23,14 +23,45 @@ import type { UiItemsProvider, Widget } from "@itwin/appui-react";
 import type { PropertyGridComponentProps } from "./PropertyGridComponent";
 
 /**
- * Id of the property grid widget created by `PropertyGridUiItemsProvider`.
+ * Props for `createPropertyGrid`.
+ * @public
+ */
+ interface PropertyGridProps {
+  /** Widget priority in the stage panel. */
+  defaultPanelWidgetPriority?: number;
+  /** Props for configuring `PropertyGridComponent` shown in the widget. */
+  propertyGridProps?: PropertyGridWidgetProps;
+}
+/**
+ * Creates a property grid definition that should be returned from `UiItemsProvider.getWidgets()`.
+ * @public
+ */
+export function createPropertyGrid({ defaultPanelWidgetPriority, propertyGridProps }: PropertyGridProps): Widget {
+  return {
+    id: "vcr:PropertyGridComponent",
+    label: PropertyGridManager.translate("widget-label"),
+    icon: <SvgInfoCircular />,
+    defaultState: WidgetState.Hidden,
+    priority: defaultPanelWidgetPriority,
+    layouts: {
+      standard: {
+        section: StagePanelSection.End,
+        location: StagePanelLocation.Right,
+      },
+    },
+    content: <PropertyGridWidget {...propertyGridProps} />,
+  };
+}
```

Comment:
Because `createPropertyGrid` changes `PropertyGridUiItemsProvider` I should write tests for it right? @saskliutas @grigasp

Reply:
I don't see much to test here - it merely returns an object. So I'd say additional tests are not necessary for this function.